sr_robot_launch/scripts/sr_ur_arm_unlock

```diff
@@ -1,5 +1,6 @@
-#!/usr/bin/python
-# Copyright 2020 Shadow Robot Company Ltd.
+#!/usr/bin/env python
```

Comment:
I'd say just put the main at the end of the library, no need to make a separate file

Reply:
the thinking here was, catkin_python won't accept a file as a python module unless it has a `.py` extension. So, to save us changing the launch files, leave this script here with the same name so it can be called the same, but move the functionality into a `.py` module